cardinal/world.go

```diff
@@ -553,6 +565,37 @@ func (w *World) UseNonce(signerAddress string, nonce uint64) error {
 	return w.redisStorage.UseNonce(signerAddress, nonce)
 }
 
+func (w *World) GetDebugState() (types.DebugStateResponse, error, error) {
```

Comment:
just call this `GetState`. The return type shouldn't be a "Response" because "Response" is a server specific concept.